server/backend/api/app/routers/router_user_role_assignment.py

```diff
@@ -24,7 +25,7 @@
 
 @router.post(
     "",
-    response_model=List[schemas.FamUserRoleAssignmentCreateResponse],
+    response_model=schemas.FamUserRoleAssignmentResponse,
```

Comment:
Hi Ian, same here, maybe you can use this schema directly as it's imported

Reply:
Removed.